edx_sga/templates/staff_graded_assignment/show.html

```diff
@@ -13,7 +13,7 @@
       <p>No file has been uploaded.</p>
     <% } %>
     <% if (graded) { %>
-      <p>Your score is <%= graded.score %> / <%= max_score %><br/>
+      <p>Your score is <%= (graded.score / max_score) * weight %> / <%= weight %><br/>
```

Comment:
@sfrucht do you know what standard edX behavior is for a problem with a weight of 0 ?